### PR TITLE
[async] Add some hyphenation hints

### PR DESCRIPTION
**Pull request by tkoeppe**

### CODE REVIEW

File: src/async.tex [hunk 1]
```diff
@@ -259,13 +259,13 @@
...
-\tcode{x1.dispatch(std::move(f),a)}  &
+\tcode{x1.dispatch(\br\mbox{\ \ }std::move(f),~a)}  &
   &
 \effects Creates an object \tcode{f1} initialized with \tcode{\textit{DECAY_COPY}(forward<Func>(f))} (\CppXref{thread.decaycopy}) in the current thread of execution . Calls \tcode{f1()} at most once. The executor may block forward progress of the caller until \tcode{f1()} finishes execution. Executor implementations should use the supplied allocator to allocate any memory required to store the function object. Prior to invoking the function object, the executor shall deallocate any memory allocated. \enternote Executors defined in this Technical Specification always use the supplied allocator unless otherwise specified. \exitnote \br
 \sync The invocation of \tcode{dispatch} synchronizes with (\CppXref{intro.multithread}) the invocation of \tcode{f1}.  \\ \rowsep
 
-\tcode{x1.post(std::move(f),a)}\br
-\tcode{x1.defer(std::move(f),a)}  &
+\tcode{x1.post(std::move(f), a)}\br
+\tcode{x1.defer(std::move(f), a)}  &
```
Comment (jwakely) [inline src/async.tex:268]: In 4a0c325ab87b60710b52fbff08293330291996e9 Lars added NBSP here instead, which doesn't make any difference to the generated PDF. Any reason to prefer one over the other?

File: src/async.tex [hunk 2]
```diff
@@ -259,13 +259,13 @@
   &
 Shall not exit via an exception. \br\textit{Precondition:} A preceding call \tcode{x2.on_work_started()} where \tcode{x1 == x2}.  \\ \rowsep
 
-\tcode{x1.dispatch(std::move(f),a)}  &
+\tcode{x1.dispatch(\br\mbox{\ \ }std::move(f),~a)}  &
```
Comment (jwakely) [inline src/async.tex:262]: This conflicts with the change in 4a0c325ab87b60710b52fbff08293330291996e9 but all that did was add the NBSP that you've also added here, so this is a better change.

**Review by jwakely — COMMENTED** [review]
Conflict resolution.

File: src/async.tex (hunk 1 above)
Comment (tkoeppe) [reply src/async.tex:268]: I guess I tried to keep the magic minimal, and this place didn't require non-breaking spaces. Note that semantically there's no reason not to break here; the only reason we do that in the place is to make _that_ custom break work correctly. Coming to think of it, perhaps we don't even need the NBSP there, either.

File: src/async.tex (hunk 2 above)
Comment (jwakely) [reply src/async.tex:262]: Committed as f0db4308b80b079baa1bf68680baf4bd874da8ac

**Comment by jwakely** [comment]
Everything addressed by this pull request has been dealt with now, sometimes as proposed and sometimes by rewording.

**Pull request closed without merging**